src/components/AppButton/AppButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactElement, PropsWithChildren } from 'react';
+
+type AppButtonVariant = 'appButton' | 'appButton_thin';
+
+export interface AppButtonProps extends PropsWithChildren {
+  variant: AppButtonVariant;
+  link: string;
+}
+
+export const AppButton = ({ variant, link, children }: AppButtonProps): ReactElement => {
```

Comment:
We have already a simple button. But the difference here is that this one is a link, call it LinkButton

Reply:
don't forget to rename all files as well